python/src/trezorlib/messages.py

```
@@ -887,16 +892,19 @@ class AptosSignTx(protobuf.MessageType):
     FIELDS = {
         1: protobuf.Field("address_n", "uint32", repeated=True, required=False),
         2: protobuf.Field("raw_tx", "bytes", repeated=False, required=True),
+        3: protobuf.Field("tx_type", "AptosTransactionType", repeated=False, required=False),
     }
```

Ruff (0.11.9)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)